unison-src/transcripts/api-getDefinition.output.md

```diff
@@ -312,99 +312,11 @@ GET /api/getDefinition?names=%23qkhkl0n238&relativeTo=nested
     },
     "typeDefinitions": {}
 }
---  Should find definitions by hash, using global names if no names in specified path.
+--  Should filter out any definitions which aren't in the provided namespace even if the hash matches.
```

Comment:
I believe these old semantics were sub-optimal anyways, if you specifically search for a hash within a specific namespace, I think you should only get hashes that are in that namespace. If you want to do a global search for a hash you can just search at the namespace root.

Reply:
I need another PR which PREFERS local names by hash, but will fall back to global names, biased towards the current perspective.

This is so you can click on external names in a pretty-printed definition.